src/Lykke.Common.Validation/StringFilters/StringFiltersExtensions.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Lykke.Common.Validation.StringFilters
+{
+    /// <summary>Provides extension methods for fitering user input strings</summary>
+    public static class StringFiltersExtensions
+    {
+        private static readonly Regex MultipleSpacesRegex = new Regex("[\\s]{1,}", RegexOptions.Compiled);
+
+        /// <summary> Default white space </summary>
+        public const char NormalSpace = '\u0020';
```

Comment:
It can be a simple string

Reply:
Made private, explicitly char code used because c# supports unicode